### PR TITLE
add /responsible-disclosure to footer

### PR DESCRIPTION
**Pull request by tati**
Add a Responsible Disclosure Program link to in-app footer https://www.thinkful.com/responsible-disclosure/

### CODE REVIEW

File: src/Footer/Footer.jsx [hunk 1]
```diff
@@ -47,6 +47,10 @@ function generateLinkSet(config) {
     {
       'name': 'Blog',
       'location': `//blog.thinkful.com`
+    },
+    {
+      'name': 'Responsible Disclosure Program',
+      'location': `${config.www.url}/responsible-disclosure`
```
Comment (diezguerra) [inline src/Footer/Footer.jsx]: no / ?

**Comment by diezguerra** [comment]
:+1: minus comments

File: src/Footer/Footer.jsx (hunk 1 above)
Comment (tati) [inline src/Footer/Footer.jsx]: none of the other ones have a trailing slash :question: @learykara?

Comment (learykara) [inline src/Footer/Footer.jsx]: they should - we use trailing slashes everywhere (except, I've noticed, ostrich)

**Comment by tati** [comment]
fixed in dif PR

**Pull request closed without merging**